### PR TITLE
Resource Article on Kubernetes Security

### PR DESCRIPTION
**Pull request by deepeshaburse**
Signed-off-by: Deepesha Burse <87636253+deepeshaburse@users.noreply.github.com>

**Description**

This PR adds a resource article on kubernetes security.

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by leecalcote** [comment]
Still a draft or ready for review?

**Comment by deepeshaburse** [comment]
@leecalcote I need to add relevant images to it but the content is ready for review.

**Comment by leecalcote** [comment]
@eeshaanSA will you aid in getting this across the finish line?

File: src/collections/resources/articles/kubernetes/simplifying-kubernetes-security/index.mdx [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+  cloud-based workloads. But how does Kubernetes' security work? </p>
+
+  <p> Kubernetes security is an important topic for businesses and organizations that rely on Kubernetes for 
+  their cloud native applications. Kubernetes is quickly becoming the standard for container orchestration 
+  and deployment, and it's important to understand the security implications of using this technology. This 
+  article explains the basics of Kubernetes security, from authentication and authorisation to network 
+  security and more. It also looks at the various tools available to help you secure your Kubernetes 
+  clusters and the best practices for keeping your environment safe. </p>
+
+  <p> The article also provides an overview of the impact of microservices and the best practices for
```
Comment (eeshaanSA) [inline src/collections/resources/articles/kubernetes/simplifying-kubernetes-security/index.mdx:33]: This seems a little repetitive. This para can be completely skipped, or maybe merged into the last paragraph. @deepeshaburse what do you think?

**Comment by eeshaanSA** [comment]
Sure

**Review by eeshaanSA — CHANGES_REQUESTED** [review]
The CTA isn't rendering, and the article is missing images. Looks good otherwise. @deepeshaburse

File: src/collections/resources/articles/kubernetes/simplifying-kubernetes-security/index.mdx (hunk 1 above)
Comment (deepeshaburse) [reply src/collections/resources/articles/kubernetes/simplifying-kubernetes-security/index.mdx]: @eeshaanSA I agree, thanks for pointing it out. I have removed that part and added it to the previous paragraph. Does it look good to you now? And, we don't have images for this topic, unfortunately.

**Comment by l5io** [comment]
🚀 Preview for commit 1d5ba8e21be1ed15b12132978a711ab31e31df93 at: https://63c8326943c898058731a26d--layer5.netlify.app

**Review by leecalcote — CHANGES_REQUESTED** [review]
Still missing images.

**Comment by deepeshaburse** [comment]
> Still missing images.

We had decided to go ahead without images.

**Review by eeshaanSA — CHANGES_REQUESTED** [review]
The CTA still isn't rendering. Is this an issue for me or? 
image
Browser: Safari

**Pull request closed without merging**